### DISCUSSION TITLE
Générez des images globallement reconnues pour vos utilisateurs avec Gravatar

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#196] [Announcements]
# Générez des images globallement reconnues pour vos utilisateurs avec Gravatar

Vos utilisateurs pourraient utiliser un seul avatar pour tout le web! Une photo chargée à un seul endroit pour une utilisation globale. Voici comment procéder gratuitement!

https://zerofiltre.tech/articles/124

**Comment by MarieSung** [comment]
<a href="http://images.google.dj/url?sa=t&url=https://chirurgieobesitetunisie.com/">chirurgie contre l'obésité</a>
